File: Praktikum_Pemrograman1-Modul 5/C/PYTHON/PRAK505-2310817320006-Alya_Rosaan.py
def Biodata(tahun, nama, asal):
    tahun_sekarang = 2023
    umur = int(tahun_sekarang) - int(tahun_lahir)
    print("perkenalkan nama saya: ", nama)
    print("umur saya: ", umur)
    print("saya adalah angkatan: ",tahun_sekarang)
    print("asal saya dari: ", asal)

tahun_lahir = int(input(""))
nama = input("")
asal = input("")
Biodata(tahun_lahir, nama, asal)

File: Praktikum_Pemrograman1-Modul 5/C/PRAK505-2310817320006-Alya_Rosaan.c
#include <stdio.h>

void Biodata(int tahun_lahir, char *namaku, char *asal){
    int tahun_sekarang = 2023;
    int umur = tahun_sekarang - tahun_lahir;
    printf("perkenalkan nama saya: %s\n", namaku);
    printf("umur saya: %d\n", umur);
    printf("saya adalah angkatan: %d\n", tahun_sekarang);
    printf("asal saya dari:: %s\n", asal);
}

int main(){
    int tahun_lahir;
        char namaku[20], asal[15];
        scanf("%d", &tahun_lahir);
        scanf("%s", &namaku);
        scanf("%s", &asal);
    Biodata(tahun_lahir, namaku, asal);
    return 0;
}